frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -476,6 +476,7 @@ const cardDurations = handleActions({
 }, {});
 
 const parameterValues = handleActions({
+    [INITIALIZE]: { next: () => ({}) },
```

Comment:
Might be worth throwing in a comment why we're setting the state to an empty object here. I was a bit confused until I looked into the underlying issue a bit more.